diff --git a/Dockerfile.core b/Dockerfile.core
index 0434d50..e0061b0 100644
--- a/Dockerfile.core
+++ b/Dockerfile.core
@@ -1,8 +1,10 @@
 FROM --platform=$BUILDPLATFORM alpine:3.15
 
+LABEL psu-version=v1.3.1
+
 RUN set -e; \
     apk add --no-cache \
-      bash ca-certificates curl gettext jq
+      bash ca-certificates curl gettext jq yq unzip wget openssh openssh-client ca-certificates httpie tzdata;
 
 ENV LANG="en_US.UTF-8" \
     LC_ALL="C.UTF-8" \
@@ -34,4 +36,4 @@ COPY psu /usr/local/bin/
 
 RUN chmod +x /usr/local/bin/psu
 
-ENTRYPOINT ["/usr/local/bin/psu"]
+# ENTRYPOINT ["/usr/local/bin/psu"]
diff --git a/psu b/psu
index b6dce0a..31f1045 100755
--- a/psu
+++ b/psu
@@ -4,7 +4,7 @@
 # List stacks, services, tasks and containers
 # And more!
 
-set -e
+#set -e
 [[ "$PSU_TRACE" ]] && set -x
 
 ############################
@@ -779,8 +779,9 @@ check_argument() {
 #   None                                  #
 ###########################################
 check_for_errors() {
+  set -e
   local exit_code=$1
-  local response=$2
+  local response="$2"
   if [ $exit_code -ne 0 ]; then
     case $exit_code in
         22)
@@ -795,6 +796,7 @@ check_for_errors() {
     esac
     exit 1
   fi
+  set +e
 }
 
 echo_debug_safe_json() {
